lib/phoenix_live_view.ex

```diff
@@ -333,6 +333,132 @@ defmodule Phoenix.LiveView do
     * `live_component` - compartmentalizes state, markup, and events
     * `live_render` - compartmentalizes state, markup, events, and error isolation
 
+  ## Uploads
```

Comment:
I suggest to move this to its own guide as we broke down the moduledoc into a bunch of guides references a couple lines below. Don't forget to add links to both chunk uploads and regular uploads in the moduledocs!

Reply:
@mcrumm